quizmaster-client-app/src/components/QuizHosting/QuestionResponse.ts

```diff
@@ -2,6 +2,7 @@ interface QuestionResponse {
   answer: string;
   name: string;
   id: string;
+  answerTime: number;
```

Comment:
Maybe change to include unit i.e. `answerTimeInSeconds`

Reply:
changed to answerTimeLeftAsAPercentage as this is what it contains